.storybook/Link.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withInfo } from '@storybook/addon-info'
+import { Link } from '../src'
+
+const description = 'A type of the <Text> component' +
```

Comment:
I'd prefer using the backpack syntax here rather than concatenating strings

Reply:
that was a typo, "backtick"

Reply:
Actually I am a fan of string literal too. Updated to use backtick. Thx.